step-capstone/src/components/Sidebars/TimeLine.js

```diff
@@ -66,9 +66,26 @@ export default class TimeLine extends React.Component {
   getIntervals() {
     let intervals = [];
     if (this.props.displayDate !== undefined) {
+      this.startOfDisplayDate = 
```

Comment:
Dangling equals? Might be a bug

Reply:
it is actually just formatting, it is on the second line for some strange reason. it gets fixed on my other PR (fixUnboundedSuggestions)